backend/ops_api/ops/utils/auth_views.py

```diff
@@ -48,17 +45,22 @@ def login() -> Union[Response, tuple[str, int]]:
 
         # See if user exists
         user = process_user(user_data)  # Refactor me
-        # user.auth_token = str(token)
-        # user.last_login = datetime.datetime.now()
-        # user.save()
 
+        # TODO
+        # Do we want to embed the user's roles or permissions in the scope: [read write]?
+
+        # The next two tokens are specific to our backend API, these are used for our API
+        # authZ, given a valid login from the prior AuthN steps above.
         access_token = create_access_token(identity=user)
         refresh_token = create_refresh_token(identity=user)
-        return jsonify(access_token=access_token, refresh_token=refresh_token)
+        response = jsonify(access_token=access_token, refresh_token=refresh_token)
+        response.headers.add("Access-Control-Allow-Origin", "*")
+        return response
 
     except Exception as err:
+        logging.error(err)
         traceback.print_exc()
-        return f"You screwed up!: {err}", 400
+        return f"Login Error: {err}", 400
```

Comment:
do we need to add the response header, i.e. `Access-Control-Allow-Origin` here as well?

Reply:
Not sure it was needed, but updating to make it consistent - we need to figure out why the global CORS isn't working - and eventually set correct origins